jobscheduler/scheduler-component-shared/src/java/org/sakaiproject/component/app/scheduler/jobs/removesites/RemoveMySitesJob.java

```diff
@@ -0,0 +1,87 @@
+package org.sakaiproject.component.app.scheduler.jobs.removesites;
+
+
+import lombok.Setter;
+import lombok.extern.java.Log;
+import lombok.extern.slf4j.Slf4j;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.sakaiproject.exception.IdUnusedException;
+import org.sakaiproject.exception.PermissionException;
+import org.sakaiproject.javax.PagingPosition;
+import org.sakaiproject.site.api.Site;
+import org.sakaiproject.site.api.SiteService;
+import org.sakaiproject.tool.api.Session;
+import org.sakaiproject.tool.api.SessionManager;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Collection;
+import java.util.List;
+
+
+/**
+ * This removes all the user's sites. This is useful if you change the template and want to have all the sites
+ * re-copied from the templates.
+ */
```

Comment:
What's this got to do with stealthing LTI tools?

Reply:
Fix fix the issue, we need to clean out all the My Workspaces because the original fix didn't work.